package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "cuttlebelle",
 	"description": "The react static site generator that separates editing and code concerns",
-	"version": "1.0.0-alpha.57",
+	"version": "1.0.0-alpha.58",
```

Comment:
Unfortunately version `1.0.0-alpha.58` has not been published yet. Slow down gunslinger 🔫 ❤️